content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -772,7 +772,7 @@ The <tt>system:node</tt> role only exists for compatibility with Kubernetes clus
 </tr>
 <tr>
 <td><b>system:node-proxier</b></td>
-<td><b>system:kube-proxy</b> user</td>
+<td><b>system:node-proxier</b> user</td>
```

Comment:
Refer to `plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go` in k/k source code.
I don't think this is correct.